deploy.sh

```diff
@@ -1,16 +1,15 @@
 #!/usr/bin/env bash
 set -xeo pipefail
 
+BUILD=$1
 
-TARGET=$1
-
-if [ -z "$TARGET" ] || [ ! -d "$TARGET" ] ; then
+if [ -z "$BUILD" ] || [ ! -d "$BUILD" ] ; then
   exit 1
 fi
 
-if [ -n "$PREVIEW_FTP_HOST" ] && [ "$PREVIEW_FTP_PATH" ] && [ -n "$PREVIEW_FTP_USERNAME" ] && [ -n "$PREVIEW_FTP_PASSWORD" ]; then
-  cd $TARGET
-  ncftpput -mR -v -u "$PREVIEW_FTP_USERNAME" -p "$PREVIEW_FTP_PASSWORD" "$PREVIEW_FTP_HOST" "$PREVIEW_FTP_PATH" ./
+if [ -n "$DEPLOY_PATH" ] && [ -d "$DEPLOY_PATH" ]; then
```

Comment:
@emiketic-admin On en a déjà parlé je pense mais j'aime pas trop ce nom de variables. On les avait pourtant retravaillées et discutée ensemble par rapport au starter kit et sa norme en général.
As-tu une argumentation spécifiques qui appuie ton choix de ce nom de variables par rapport à ce que j'avais proposé la dernière fois et qui a été supprimé ici en rouge ?

Reply:
it's my fault, I copied some files from wensh project without review.
I fixed it